ddtrace/_trace/tracer.py

```diff
@@ -549,9 +549,9 @@ def configure(
                 sync_mode=self._use_sync_mode(),
                 api_version=api_version,
                 # if apm opt out, neither agent or tracer should compute the stats
-                headers={"Datadog-Client-Computed-Stats": "yes"}
-                if (compute_stats_enabled or self._apm_opt_out)
-                else {},
+                headers=(
+                    {"Datadog-Client-Computed-Stats": "yes"} if (compute_stats_enabled or self._apm_opt_out) else {}
+                ),
```

Comment:
Do we need to make this change?

Reply:
I believe the formatting command was responsible for this. Should it be reverted?

Reply:
Nope that's fine